src/app/components/VisualImage.js

```diff
@@ -0,0 +1,26 @@
+function VisualImage(imageSource) {
+  // Checking that img source is in base64 format
+  // FIXME: If we should use only base64/jpg, delete one condition in if
+  const reg = /^data:image\/(?:gif|png|jpeg|bmp|webp)(?:;charset=utf-8)?;base64,(?:[A-Za-z0-9]|[+/])+={0,2}/;
+  if (reg.test(imageSource) || /.jpg/.test(imageSource)) {
+    // Creating div and img
+    const createDiv = document.createElement('div');
+    createDiv.className = 'visual-box';
+
+    const createImg = document.createElement('img');
+    createImg.classList.add('visual-box__image');
+    createImg.setAttribute('alt', 'image');
+
+    // Placing div with img inside container with id = "swquiz-app"
+    document.querySelector('#swquiz-app').appendChild(createDiv);
+    document.querySelector('.visual-box').appendChild(createImg);
+
+    // Adding sourse to img
+    const img = document.querySelector('.visual-box__image');
```

Comment:
Instead of new querySelector, set the attribute on 'createImg' element.
`createImg.setAttribute('src', imageSource);`
After that you should return the created component.